config/trainer_config.yaml

```diff
@@ -137,9 +137,10 @@ Tennis:
     time_horizon: 1000
     self_play:
         window: 10
-        play_against_current_self_ratio: 0.5
+        play_against_latest_model_ratio: 0.5
```

Comment:
Need to mention the parameter rename in the MigrationGuide (or check for the old name).

Reply:
Added to the migration doc